### ISSUE TITLE
Material ReDesign 

### ISSUE THREAD

**Issue by adarshv90** [prabhat90/DownloadManager#47]
Spectacular extension. Please issue a Material Redesign to go inline with Google's design language

**Comment by austin-guevara** [comment]
@adarshv90 the first step would be mockups! I would love to start some and would encourage anyone else who's interested to join in as well.

**Comment by austin-guevara** [comment]
Just realized that the last commit on this repository was 2 years ago 😞

**Comment by adarshv90** [comment]
@austin-guevara Ya I saw that after I posted the request as well. I've reverted to default Chrome Download Manager 😔. Still on the lookout for a good replacement

**Comment by austin-guevara** [comment]
@adarshv90

The quality of the native Google Chrome "Downloads" tab is really excellent. I just can't stand how Chrome always opens the bar at the bottom of my window that I always have to dismiss.

I'm working on my own extension now, as I haven't been able to find anything that suits my needs. It will open up the Chrome Downloads tab instead of the bottom bar. I'll update this thread if I'm able to actually make any progress (as I've never created a Chrome extension) 😄

**Comment by adarshv90** [comment]
@austin-guevara 

I agree completely. That's my only gripe with the default implementation. That and there is no way of knowing download progress if the bottom bar is dismissed.

I hate to draw comparisons, but I prefer the way downloads are handled in Firefox (hence the search for good extensions).

Good luck with your project. I'd be glad to provide any design related assistance or guidance.
